### DISCUSSION TITLE
Statiq is not working for me

### DISCUSSION THREAD

**Discussion by JoshuaKrimmer** [statiqdev/Discussions#224] [Q&A]
I have created a sample project called "Dokumentation" and installed the latest Version of Statiq.Docs 1.0.0-beta.17. For testing purposes, I created a class "Class1" and gave it some XML documentation. The Program.cs contains the setup process as instructed in the Quick Start Guide.

![grafik](https://github.com/user-attachments/assets/b55c874f-9d6d-44d0-8356-398cb6e5deb2)
![grafik](https://github.com/user-attachments/assets/93975cad-3008-4d04-98dc-c31344d666d7)

When I have understood the documentation correctly, the default search path is the input folder, that is why I added the document of "Class1.cs" as a fixed Source file. After running the project, the Console has the following output, and it seems the file was recognized properly.

![grafik](https://github.com/user-attachments/assets/dbc7ede7-aa4e-4300-9776-9f0c6f309ef2)

But for some reason, the output only contains the sitemap.xml file and nothing else.

[log.txt](https://github.com/user-attachments/files/16311448/log.txt)

Thank you very much for your help!

**Comment by girlpunk** [comment]
As well as the source code, have you given Statiq a theme to use?
For your use case, you may want to start with [Docable](https://github.com/statiqdev/docable)
